backend/request.ts

```diff
@@ -904,23 +911,23 @@ export async function parseRemoveAssigneeRequest(
 
 export async function parseRemoveCoachRequest(
     req: express.Request
-): Promise<Requests.RmDraftCoach> {
-    return hasFields(req, ["project_user"], types.id).then(() =>
+): Promise<Requests.Coach> {
+    return hasFields(req, ["loginUserId"], types.id).then(() =>
```

Comment:
why the name change?

Reply:
This parser used to expect a project_user_id, but the logic of removing a coach was changed to work with a project_id and a login_user_id so the parser expects a login_user_id now.

Reply:
Hmmm okay